### ISSUE TITLE
Training on WIDER data for face detection

### ISSUE THREAD

**Issue by mengzhangjian** [rbgirshick/py-faster-rcnn#300]
I  try to do face detection using faster-RCNN with WIDER data, I made some modifications in lib/datasets/pascal_voc.py:

  for i in range(num_objs):
                #bbox = obj.find('bndbox')
                # Make pixel indexes 0-basedn
                x1 = float(split_line[2+i*4]) 
                y1 = float(split_line[3+i*4]) 
                x2 = float(split_line[4+i*4]) 
                y2 = float(split_line[5+i*4])
                cls = self._class_to_ind['face']
                boxes[i, :] = [x1, y1, x2, y2]
                gt_classes[i] = cls
                overlaps[i, cls] = 1.0
                seg_areas[i] = (x2 - x1 + 1) * (y2 - y1 + 1)
The WIDER data format is like this:
/home/zj/py-faster-rcnn/data/WIDERdata/WIDER_train/images/0--Parade/0_Parade_marchingband_1_849 1 449 330 570 478
1 is the number of face object, 449 330 570 478 is (left,top,right,bottom)

I also change the num_classes in *.pt file,but when the training is on the stage1 fast rcnn , the bbx_loss is nan, I decrease the learning rate to 0.0001,but it help nothing, can anyone help me fix this or tell me what should i do ?

**Comment by george-adaimi** [comment]
@mengzhangjian 
Hey,

I am facing a problem to find how to train faster R-CNN on WIDER dataset. Can you help me out with it? Were you able to train it? If not, were you able to find a pre-trained model on WIDER dataset?

I want to integrate faster R-CNN to detect a face and then provide this face to a face recognition algorithm?

I would really appreciate your help.
